cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -229,6 +250,36 @@ inline fun <reified T : Statement> FunctionDeclaration.body(n: Int = 0): T {
 
 class StatementNotFound : Exception()
 
+class NodePath(var nodes: List<Node>, var assumptions: MutableList<Assumption>) {
```

Comment:
could be a data class, also documentation missing

Reply:
It was in the past, before i added the assume function. Now, with the interface i may be able to remove it again and make it a dataclass.

Reply:
I removed NodePath again. If we properly use List<Edge> instead of List<Node> the information is already reachable over the edge.